### PR TITLE
Adding keyword traffic estimation examples.

### PR DESCRIPTION
**Pull request by mcloonan**
Change-Id: Ie204163ad61a05ff61d1ef129b025e391b0d6c6d

### CODE REVIEW

File: examples/planning/add_keyword_plan.rb [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+def add_keyword_plan(customer_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  keyword_plan = create_keyword_plan(client, customer_id)
+  plan_campaign = create_keyword_plan_campaign(
+    client,
+    customer_id,
+    keyword_plan
```
Comment (samphippen) [inline examples/planning/add_keyword_plan.rb]: not to be "that guy" but I generally prefer style where you leave a trailing comma on a line broken call.

File: examples/planning/add_keyword_plan.rb [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+end
+
+def create_keyword_plan(client, customer_id)
+  keyword_plan = client.resource(:KeywordPlan)
+  keyword_plan.name = client.wrapper.string(
+    "Keyword plan for traffic estimate ##{(Time.new.to_f * 1000).to_i}"
+  )
+  forecast_period = client.resource(:KeywordPlanForecastPeriod)
+  forecast_period.date_interval =
+      client.enum(:KeywordPlanForecastInterval)::NEXT_QUARTER
```
Comment (samphippen) [inline examples/planning/add_keyword_plan.rb]: in all my examples I use the symbol literal e.g. `:NEXT_QUARTER`. Can we standardize on one style?

**Comment by samphippen** [comment]
I came here to be the bad style review human. Structurally this looks good to me so feel free to merge!

File: examples/planning/add_keyword_plan.rb (hunk 2 above)
Comment (mcloonan) [reply examples/planning/add_keyword_plan.rb]: Yes, now that I know this is possible it's clearly better.

**Pull request merged**